From 6b678cb714197448c63fb170cfc8a07cf3242e62 Mon Sep 17 00:00:00 2001
Message-ID: <6b678cb714197448c63fb170cfc8a07cf3242e62.1694948061.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1694948061.git.sharlatanus@gmail.com>
References: <cover.1694948061.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 17 Sep 2023 11:51:00 +0100
Subject: [PATCH 2/2] gnu: Sort use-module in (gnu packages virtualization)
 alphabetically.

* gnu/packages/virtualization.scm. Sort use-module alphabetically to
easy maintenance.
---
 gnu/packages/virtualization.scm | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/gnu/packages/virtualization.scm b/gnu/packages/virtualization.scm
index 04ed253e8a..f30ed75433 100644
--- a/gnu/packages/virtualization.scm
+++ b/gnu/packages/virtualization.scm
@@ -81,8 +81,8 @@ (define-module (gnu packages virtualization)
   #:use-module (gnu packages fonts)
   #:use-module (gnu packages fontutils)
   #:use-module (gnu packages freedesktop)
-  #:use-module (gnu packages gettext)
   #:use-module (gnu packages gcc)
+  #:use-module (gnu packages gettext)
   #:use-module (gnu packages gl)
   #:use-module (gnu packages glib)
   #:use-module (gnu packages gnome)
@@ -115,17 +115,17 @@ (define-module (gnu packages virtualization)
   #:use-module (gnu packages pkg-config)
   #:use-module (gnu packages polkit)
   #:use-module (gnu packages protobuf)
+  #:use-module (gnu packages pulseaudio)
   #:use-module (gnu packages python)
   #:use-module (gnu packages python-build)
   #:use-module (gnu packages python-check)
   #:use-module (gnu packages python-crypto)
   #:use-module (gnu packages python-web)
   #:use-module (gnu packages python-xyz)
-  #:use-module (gnu packages pulseaudio)
   #:use-module (gnu packages readline)
   #:use-module (gnu packages rsync)
-  #:use-module (gnu packages selinux)
   #:use-module (gnu packages sdl)
+  #:use-module (gnu packages selinux)
   #:use-module (gnu packages sphinx)
   #:use-module (gnu packages spice)
   #:use-module (gnu packages ssh)
@@ -137,6 +137,7 @@ (define-module (gnu packages virtualization)
   #:use-module (gnu packages xdisorg)
   #:use-module (gnu packages xml)
   #:use-module (gnu packages xorg)
+  #:use-module ((guix licenses) #:prefix license:)
   #:use-module (guix build-system cmake)
   #:use-module (guix build-system gnu)
   #:use-module (guix build-system go)
@@ -144,9 +145,8 @@ (define-module (gnu packages virtualization)
   #:use-module (guix build-system python)
   #:use-module (guix build-system trivial)
   #:use-module (guix download)
-  #:use-module (guix git-download)
   #:use-module (guix gexp)
-  #:use-module ((guix licenses) #:prefix license:)
+  #:use-module (guix git-download)
   #:use-module (guix packages)
   #:use-module (guix utils)
   #:use-module (srfi srfi-1)
-- 
2.41.0

